### ISSUE TITLE
[Supporting the platform - Server request>Inspect]: Controls present under "Query" tab are not visible after setting the view port at 320*256.

### ISSUE THREAD

**Issue by natellium** [grafana/grafana#66585]
## User Experience: 
Low vision users who depend on zooming the page for better readability will be impacted as they will not be able to see the complete content present in the page if the controls are not visible or truncated which will result in lack of understandability and will fail to access the controls. 

## Pre-requisite:  
Achieve viewport 320 X 256 by adjusting the browser. (Open developer's tool, resize and zoom the browser to achieve viewport 320 X 256).

## Repro Steps:
Open #URL: https://play.grafana.org/. Sign in with valid credentials. "Home" page appears. Navigate the "Server Requests Graphs". Expand the button and select the "Inspect" button. Select "Data" menu item. Navigate all the controls under the "Query" tab pane. Select the "Refresh" button. Select "+Expand all" button. Navigate to "Object " section. Verify the issue.
 Actual Result: Controls present under "Query" tab are not visible after setting the view port at 320*256.

## Expected Result:
 Controls such as "Query" "Refresh" button, "Object", "request: object" and "response: Array [4]" present under "Query " tab should be clearly visible after setting the view port at 320*256.

## Severity
`SEV2`
## Area
`TestRuleResult Component (Alerting Inspect Drawer) `
